docs/fuel-101/understanding-the-bridge.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Understanding The Bridge
+category: Fuel 101
+---
+
+# Understanding The Bridge
+
+## What is a bridge?
+
+Different blockchain protocols cannot communicate with each other directly without a special mechanism to work together. A bridge is a mechanism that allows just that: bridges allow different blockchains to communicate with each other.
+
+Fuel uses a bridge to connect the Ethereum blockchain with the Fuel blockchain. In the future, Fuel could be connected to a number of different blockchains using bridges.
+
+## Why would someone need a bridge?
+
+Bridges allow users to move assets between different blockchains. For example, if you own ETH on the Ethereum blockchain, you can use a bridge to move it to the Fuel blockchain.
+
+This allows users to take advantage of faster execution and cheaper transaction on the Fuel network using the same assets they already own on another network. It also allows developers to build cross-chain applications, that work with assets on multiple blockchains.
+
+## How does the Fuel bridge work?
+
+The Fuel bridge utilizes smart contracts on both the Ethereum and Fuel networks.
+
+Once a bridge is available between two blockchains, assets and data from one network are not automatically available on the other. Users must initiate the bridging process. Fuel provides a convenient [UI](https://next-app.fuel.network/bridge) to make this process easy for users, but it is possible to use the bridge without a UI as well.
+
+## What types of things can be bridged?
+
+You can bridge anything! The UI currently only supports bridging ETH & ERC-20 tokens, but the underlying bridge architecture can support the bridging of any type of asset or data.
+
+## How does a deposit work?
+
+The process for depositing, or bridging something from Ethereum to Fuel, is slightly different depending on whether or not you are bridging ETH.
+
+For any deposit:
+
+1. A user initiates the deposit on the bridge contract on Ethereum.
+2. An event is emitted in the transaction that contains information about the deposit.
+3. The transction is confirmed and gets included in a block on Ethereum.
+4. A Fuel node (`fuel-core`) picks up the event from the Fuel message portal. If the user is depositing ETH, the node will mint the ETH to the destination wallet on Fuel and the process is complete. If the user is depositing anything besides ETH:
+5. The Fuel node will generate a message UTXO and sends it to the user's address on Fuel.
```

Comment:
Small nit, but not sure how I would put this.

The UTXO is sent (ie. its ownership is assigned) to a predicate that validates the correctness of the bridging operation and allows anyone to relay it to the L2 bridge 's`process_message`, which in turns mints the final UTXO containing the "mirror" asset of the deposit that happened at L1.